packages/angular/test/base/e2e/src/router-link.spec.ts

```diff
@@ -113,6 +113,7 @@ describe('Router Link', () => {
   describe('back', () => {
     it('should go back with ion-button[routerLink][routerDirection=back]', () => {
       cy.get('#routerLink-back').click();
+      testBack();
```

Comment:
I don't understand the test change here. The fix modifies whether or not an animation is used, but `testBack` does not verify this.

Reply:
This was just an issue I fixed where the test executed the back navigation, but did not call `testBack` as the other tests. I assumed it was missed originally when writing the test.

Reply:
Thanks for the clarification. Is this change necessary to validate the bug you fixed? If not, could we remove it from the PR?

Reply:
Thanks for the clarification. Is this change necessary to validate the bug you fixed? If not, could we remove it from the PR?

Reply:
Thanks for the clarification. Is this change necessary to validate the bug you fixed? If not, could we remove it from the PR?

Reply:
Sure :)